pkg/services/publicdashboards/api/api.go

```diff
@@ -252,3 +299,66 @@ func toJsonStreamingResponse(features *featuremgmt.FeatureManager, qdr *backend.
 
 	return response.JSONStreaming(statusCode, qdr)
 }
+
+// swagger:response listPublicDashboardsResponse
+type ListPublicDashboardsResponse struct {
+	// in: body
+	Body PublicDashboardListResponseWithPagination `json:"body"`
+}
+
+// swagger:parameters getPublicDashboard
+type GetPublicDashboardParams struct {
+	// in:path
+	DashboardUid int64 `json:"dashboardUid"`
```

Comment:
this should be `string`, no? Same for the POST and PUT parameter

Reply:
Good catch, I changed `uids` too. `DashboardUid` and `Uid` are strings.